src/services/scheduler.ts

```diff
@@ -83,7 +83,13 @@ export async function start(dbId: string, dbUser:string) {
         } catch (e) {
           logUserErr(e);
           // error must be valid JSON as a string
-          const error = JSON.stringify(e, Object.getOwnPropertyNames(e));
+          const error = JSON.stringify(e, (_key, value) => {
+            if (Array.isArray(value)) {
+              return value.map(val => JSON.stringify(val, Object.getOwnPropertyNames(val)))
+            } else {
+              return value
+            }
+          });
```

Comment:
I'd recommend turning this into a helper function and on the `if (Array.isArray(value))` path, it calls itself (to handle multiple-levels of array nesting.

Reply:
for the recursiveness had to take inspiration from [here](https://www.bennadel.com/blog/3278-using-json-stringify-replacer-function-to-recursively-serialize-and-sanitize-log-data.htm) because it was not working. If you just return the `Error` it appear as an empty object in the string.